src/pages/project/tests/projectContributorHelpers.test.ts

```diff
@@ -2,41 +2,202 @@ import { describe, expect, it } from 'vitest';
 import { addContributor, AddContributorErrors } from '../helpers/projectContributorHelpers.js';
 import {
   abcOrgAsAffiliation,
+  contributorsArrayWithDifferentProjectManagerWithDifferentAffiliation,
+  contributorsArrayWithDifferentProjectManagerWithSameAffiliation,
+  contributorsArrayWithDifferentProjectManagerWithUndefinedAffiliation,
+  contributorsArrayWithOtherPersonWithDifferentAffiliation,
+  contributorsArrayWithOtherPersonWithSameAffiliation,
+  contributorsArrayWithOtherPersonWithUndefinedAffiliation,
   contributorsArrayWithProjectManager,
+  contributorsArrayWithSelectedPersonAsProjectManagerWithDifferentAffiliation,
+  contributorsArrayWithSelectedPersonAsProjectManagerWithNoAffiliation,
+  contributorsArrayWithSelectedPersonAsProjectManagerWithSameAffiliation,
+  contributorsArrayWithSelectedPersonWithDifferentAffiliation,
+  contributorsArrayWithSelectedPersonWithSameAffiliation,
+  contributorsArrayWithSelectedPersonWithUndefinedAffiliation,
+  defOrgAsAffiliation,
+  existingPersonIdentity,
   selectedPersonIdentity,
   selectedPersonWithAffiliation,
   selectedPersonWithoutAffiliation,
 } from './mockObjects';
 
 describe('addContributor', () => {
   describe('when adding a project manager with affiliation', () => {
-    it.skip('if there is already a project manager in the contributors array it returns an error', async () => {
+    it('if there is already a project manager in the contributors array it returns an error', async () => {
```

Comment:
Ser at `async` brukes for alle `it`-metoder her, men siden ingen har noen `await` vil jeg tro de er overflødige 
```suggestion
    it('if there is already a project manager in the contributors array it returns an error', () => {
```